src/CircleEditor/CircleEditorDocument.ts

```diff
@@ -17,9 +17,19 @@
 import * as flatbuffers from 'flatbuffers';
 import * as vscode from 'vscode';
 
+import {Balloon} from '../Utils/Balloon';
 import {Disposable} from '../Utils/external/Dispose';
 
 import * as Circle from './circle_schema_generated';
+import * as Types from './CircleType';
+
+/**
+ * Make BigInt data to string type.
+ * This is called by JSON.stringify function.
+ */
+(BigInt.prototype as any).toJSON = function() {
```

Comment:
Quite interesting... does this have an effect only on this file? or imported file too?